static/js/modules/typeahead.js

```diff
@@ -52,8 +53,13 @@ module.exports = {
         url = "/rest/names?q=%QUERY";
 
     if (typeof API_LOCATION !== 'undefined') {
-        url = "/names?q=%QUERY&api_key=6wpqET5GjxCwAyBqXjc4dWrfMdshFE6wu3txb6yZ"; // Replace with the correct api key
-        url = API_LOCATION.concat(url);
+        url = URI(API_LOCATION)
+          .path([API_VERSION, 'names'].join('/'))
+          .query({
+            q: '%QUERY',
+            api_key: '6wpqET5GjxCwAyBqXjc4dWrfMdshFE6wu3txb6yZ'
```

Comment:
is api_key supposed to be a constant here? Shouldn't it be reading it from config somehow?